### ISSUE TITLE
not getting good results

### ISSUE THREAD

**Issue by taghizad3h** [yaushian/CycleGAN-sentiment-transfer#1]
Hi

first, I  have to say thanks for your clean implementation.

I just cloned your repo and trained it but the results were not satisfying.
I know you have putted a trained model and it works fine, but I couldn't get good results when I trained model myself and I saw repeatedly same words on output.
So is there any configuration we have to set?
How many steps did you train?
How much time did it take?

**Comment by beyhangl** [comment]
@yaushian I couldn't get same results too. how did you solve this problem?

**Comment by skwolvie** [comment]
Hi, 

In my case, the model pre-trained for 2 straight days and still didn't stop, I stopped it by hitting ctrl+c.

Later, the "train -mode all" also ran roughly for 2 days, when I revisited the logs, I see that the training loss was NaN after five steps. So possibly the learning rate is bad.

I don't see the model training stopping on its own with early stopping. 

Additionally. I input anything, to the final model, it gives me the same output as given below:
¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸« ¸«

Can I get any insights into the ideal learning rate, and runtime of this project? 4 days of runtime achieved nothing.
